### PR TITLE
Hosted Video 

### PR DESCRIPTION
**Pull request by rich-nguyen**
I'm reverting #13616 because we don't think this is the root cause. And moving `host-about` init to commercial from enhance, and tweaking the init test so it checks the tone, not the content type. 

This fixes the about button, and shouldn't affect the video upgrade.

### CODE REVIEW

**Comment by kelvin-chappell** [comment]
As far as I can tell! 👍

File: static/src/javascripts/projects/common/modules/commercial/hosted-video.js [hunk 1]
```diff
@@ -85,101 +87,105 @@ define([
...
-                    if (vol) {
-                        player.volume(0);
-                        player.volume(vol);
+        return new Promise(function (resolve) {
+            require(['bootstraps/enhanced/media/main'], function () {
+                require(['bootstraps/enhanced/media/video-player'], function (videojs) {
+                    var $videoEl = $('.vjs-hosted__video');
+
+                    if (!$videoEl.length) {
+                        return;
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/hosted-video.js]: Should be `resolve()` instead.

File: static/src/javascripts/projects/common/modules/commercial/hosted-video.js [hunk 2]
```diff
@@ -85,101 +87,105 @@ define([
...
-                    upgradeVideoPlayerAccessibility(player);
-
-                    // unglitching the volume on first load
-                    vol = player.volume();
-                    if (vol) {
-                        player.volume(0);
-                        player.volume(vol);
+        return new Promise(function (resolve) {
+            require(['bootstraps/enhanced/media/main'], function () {
+                require(['bootstraps/enhanced/media/video-player'], function (videojs) {
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/hosted-video.js]: Man this function is looooooooooong. Might be worth abstracting a little in another PR 😮

**Comment by regiskuckaertz** [comment]
Aside from minor comment, gooooo 👍

File: static/src/javascripts/projects/common/modules/commercial/hosted-video.js (hunk 1 above)
Comment (rich-nguyen) [inline static/src/javascripts/projects/common/modules/commercial/hosted-video.js]: yep good spot

**Comment by Calanthe** [comment]
👍 
I will make a card for myself to extract some of the code in the`hosted-video.js` file.
